setup.py

```diff
@@ -28,7 +28,9 @@
         'pathos == 0.2.4',
         'pillow',
         'psutil',
-        'pybullet == 2.5.0',
+        #'pybullet == 2.5.0',
```

Comment:
For some reason, if readthedocs tries to install pybullet, there will be an error. And because pybullet is huge and rarely used by most training, here I change it to be installed manually by users.